drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -221,6 +223,120 @@ static const struct {
 };
 BUILD_ASSERT(ARRAY_SIZE(gpio_irqs) == IT8XXX2_IRQ_COUNT + 1);
 
+/* gpio group a, b, c, d, e, f, g, h, i, j, k ,l ,m, ksi, ksoh, and ksol */
+#define GPIO_GROUP_COUNT 16
+#define GPIO_GROUP_INDEX(label) DT_PROP(DT_NODELABEL(label), index)
+
+/* general control registers for selecting 1.8V/3.3V */
+static const struct {
+	volatile uint8_t *reg_1p8v;
+	uint8_t mask_1p8v;
+} gpio_1p8v[GPIO_GROUP_COUNT][8] = {
+	[GPIO_GROUP_INDEX(gpioa)] = {
+		[4] = {&IT8XXX2_GPIO_GCR24, BIT(0)},
```

Comment:
Why use addresses here? Wouldn't it be better to store the index, then get the address at runtime?

Reply:
Done